scripts/hash_file.ts

```diff
@@ -0,0 +1,13 @@
+import { createHash } from 'crypto';
+import { readFile } from 'fs/promises';
+
+async function getFileHash(path: string): Promise<string> {
+    const fileData = await readFile(path);
+    let h = createHash('sha256');
+    h.update(fileData);
+    const result = h.digest('hex');
+    console.log(result);
```

Comment:
done

Reply:
Note: we do need the output because this script is principally called from the command line. But I updated it to info and moved it out of the function